README.md

```diff
@@ -1,3 +1,47 @@
-# Typing Indicator Plugin
+# Typing Indicator Plugin for Chat Engine
+
+Adds the ability to show a typing indicator on a ChatEngine Chat
+
+### Quick Start
+
+0. Have a ChatEngine server running already, instantiate a client and connect it
+```js
+const ChatEngine = ChatEngineCore.create({
+    publishKey: 'pub-key-here',
+    subscribeKey: 'sub-key-here'
+}, {
+    endpoint: 'http://chatengine:server/',
+    globalChannel: 'global-channel-name'
+});
+
+ChatEngine.connect('Username');
+ChatEngine.on('$ready', () = { ... });
```

Comment:
I think eventually we'll be able to stop talking about $.ready event, much like how jQuery doesn't always list it because it's assumed that you know to do this.